tests/core/test_formatters.py

```diff
@@ -10,12 +14,42 @@ def test_formatter_00(self):
         f = Formatter()
         assert f._get_formatter()
 
+    def test_formatter_count(self, capsys):
+        from sys import stdout
+        f = CountFormatter(output=stdout)
+        f.printer(RESULTS)
+        out, _ = capsys.readouterr()
+        assert out.strip() == "1"
+
+    def test_formatter_csv(self, capsys):
+        """Ensure some correct text is in the output."""
+        from sys import stdout
+        f = CSVFormatter(output=stdout)
+        f.printer(RESULTS)
+        out, _ = capsys.readouterr()
+        assert "2005-12-05" in out
```

Comment:
This test should fail as the date does not have the right format. We expect the date to be `MM/DD/YYYY`.